### PR TITLE
Create RA_Test-File_Eins.md

### PR DESCRIPTION
**Pull request by rh851**
### Test results

No Test yet

### Who should know about the change?

Described in #8.

### CODE REVIEW

File: RA_Test-File_Eins.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Try **italics** and *bold* at once!
```
Comment (GH-mv) [inline RA_Test-File_Eins.md]: Yes that the first try go ahead

**Review by GH-mv — DISMISSED** [review]
Add mor e as one line

File: RA_Test-File_Eins.md (hunk 1 above)
Comment (RobinBreyl) [inline RA_Test-File_Eins.md]: Looks great !

**Comment by if338** [comment]
fine for me

**Comment by Jessehaslam1983** [comment]
Hey

**Pull request merged**